include/InlineEditing/InlineEditing.php

```diff
@@ -128,6 +128,7 @@ function getEditFieldHTML($module, $fieldname, $aow_field, $view = 'EditView', $
     $validMultipleElementNames = array(
```

Comment:
Going to look into removing "group" from email1 to remove the issue we have with emails when not using a comparison array to check for multiple fields as the "group" in email1 appears to be legacy. If so, I'll remove the comparison array and the "group" key from the email1 array in vardefs. :)